src/models/piranha_arguments.rs

```diff
@@ -61,172 +68,237 @@ pub struct PiranhaArguments {
   #[clap(short = 'f', long)]
   #[serde(skip)]
   path_to_configurations: String,
+
   /// Path to output summary json file
   #[get = "pub"]
   #[builder(default = "default_path_to_output_summaries()")]
   #[clap(short = 'j', long)]
   #[serde(skip)]
   path_to_output_summary: Option<String>,
 
-  // a list of file extensions
-  // #[get = "pub"]
-  #[builder(default = "default_languages()")]
+  // the target language
+  #[builder(default = "default_language()")]
   #[clap(skip)]
-  #[serde(default = "default_languages")]
-  language: Vec<String>,
+  #[serde(default = "default_language")]
+  language: String,
 
   #[get = "pub"]
   #[builder(default = "default_piranha_language()")]
   #[clap(skip)]
   #[serde(skip)]
   piranha_language: PiranhaLanguage,
 
-  // User option that determines whether an empty file will be deleted
+  /// User option that determines whether an empty file will be deleted
```

Comment:
Why are some of these `///` rather than `//`? Does that have an special meaning in Rust comments?

Reply:
`//` means a single line comment 
`///` is used for documenting a method/field/class 

Standard libraries that use documentation to create messages (for instance `clap` our command line parsing tool) recognize `///` for getting comments associated to elements.